README.md

```diff
@@ -360,6 +360,58 @@ func main() {
 }
 ```
 
+### Driver Options
+
+Configuration options can be provided by the standard DSN (Data Source Name).
+
+```
+[user[:password]@]addr[/db[?param=X]]
+```
+
+#### `compress`
+
+Enable zlib compression between the client and the server.
+
+| Type      | Default   | Example                                     |
+| --------- | --------- | ------------------------------------------- |
+| boolean   | false     | user:pass@localhost/mydb?compress=true      |
```

Comment:
I think it might be better to have something like `compress=zlib` / `compress=zstd` to allow selection of the compression algorithm.

MySQL Client has deprecated `--compress` in favor of `--compression-algorithms=` and `--zstd-compression-level=`.

```
$ mysql --help
...
  -C, --compress      Use compression in server/client protocol.
...
  --compression-algorithms=name 
                      Use compression algorithm in server/client protocol.
                      Valid values are any combination of
                      'zstd','zlib','uncompressed'.
  --zstd-compression-level=# 
                      Use this compression level in the client/server protocol,
                      in case --compression-algorithms=zstd. Valid range is
                      between 1 and 22, inclusive. Default is 3.
```